cmd/plasmacli/subcmd/eth/challenge.go

```diff
@@ -82,7 +70,7 @@ Usage:
 		var txBytes, proof, confirmSignatures []byte
 		if viper.GetBool(useNodeF) {
 			var result *tm.ResultTx
-			result, confirmSignatures, err = getProof(ctx, owner, challengingPos)
```

Comment:
why was the context parameter removed? I think each command should be responsible for constructing it's context object with the plasmad node